---
title: useAuthError
description: Overview of the useAuthError function
slug: wallets/reference/account-kit/react/src/functions/useAuthError
layout: reference
---

```ts
function useAuthError(): UseAuthErrorResult;
```

Defined in: [account-kit/react/src/hooks/useAuthError.ts:21](https://github.com/alchemyplatform/aa-sdk/blob/da19afd63b0ba06a4de95b0d2d7871f64d546304/account-kit/react/src/hooks/useAuthError.ts#L21)

Returns the error returned from the current auth step, if it exists

## Example

```ts twoslash
import { useAuthError } from "@account-kit/react";

const error = useAuthError();

if (error) {
  console.error("Error occurred during auth step", error);
}
```

## Returns

[`UseAuthErrorResult`](../type-aliases/UseAuthErrorResult.mdx)

the current Error object
